main.py

```diff
@@ -8,8 +8,9 @@ def main():
     gif = Gif()
     gif.make_gif(clip=clip, start_time=(0, 24), end_time=(0, 27))
     gif.resize(new_size=0.6)
-    # gif.clip = gif.clip.crop(x1=60, x2=300)
-    gif.clip.write_gif('test1.gif')
+    gif.add_text(text='In my nightmares\nI see rabbits.', font_size=30, color='white',
```

Comment:
Can a gif be written without adding any text? I'm guessing you can, but should be added as tests.

Reply:
Yes.